src/xrpld/app/tx/detail/Escrow.cpp

```diff
@@ -97,9 +98,28 @@ EscrowCreate::makeTxConsequences(PreflightContext const& ctx)
     return TxConsequences{ctx.tx, ctx.tx[sfAmount].xrp()};
 }
 
+XRPAmount
+EscrowCreate::calculateBaseFee(ReadView const& view, STTx const& tx)
```

Comment:
How is ComputeFee collected at EscrowFinish?

Since the InstructionCount is unknown before Wasm execution, do we specify maximum ComputeFee that the user pays in the transaction field?

Reply:
TBD exactly how fees will be handled. It's a constant 1000 drops for now.

As noted in the PR description, this is a very early version of the code, and improvements will continue to be made in future PRs.